terraform/cloudfoundry/security-groups.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_security_group" "web" {
+  name = "${var.env}-web-cf"
```

Comment:
The name is `${var.env}-web-cf` but the tag below is `${var.env}-cf-web`. 

Can you change both to `"${var.env}-cf-web"` as other resources are called `${var.env}-cf-*`